recipes-bsp/optee-os/optee-os-iot2050_3.16.0_rpmbw.bb

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Ivan Mikhaylov <ivan.mikhaylov@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require optee-os-iot2050_3.16.0.inc
+
+OPTEE_EXTRA_BUILDARGS += " CFG_RPMB_WRITE_KEY=y"
```

Comment:
Let's control that via an `OVERRIDE`, something like `OPTEE_EXTRA_BUILDARGS_append_rpmb-setup = '...'`. Then you can fold both recipes together again. The override would be set by a kas option file, similar to the secure boot enabling.

Reply:
ok, I'll check how it will goes with OVERRIDE.